debian/control

```diff
@@ -2,8 +2,8 @@ Source: hildon-application-manager
 Section: misc
 Priority: optional
 Maintainer: Marius Vollmer <marius.vollmer@nokia.com>
-Build-Depends: debhelper (>= 4.0.0), libapt-pkg-dev (>= 0.7.6), libglib2.0-dev, libgtk2.0-dev, libhildon1-dev, libhildonfm2-dev, libconic0-dev, libgconf2-dev, libgnomevfs2-dev, mce-dev, libhildondesktop1-dev, libalarm-dev, libtime-dev, osso-af-settings, libcurl4-openssl-dev, maemo-launcher-dev, maemo-system-services-dev
-Standards-Version: 3.6.0
+Build-Depends: debhelper (>= 9), libapt-pkg-dev (>= 0.7.6), libglib2.0-dev, libgtk2.0-dev, libhildon1-dev, libhildonfm2-dev, libconic0-dev, libgconf2-dev, mce-dev, libhildondesktop1-dev, libalarm-dev, libtime-dev, osso-af-settings, libcurl4-openssl-dev, maemo-launcher-dev, maemo-system-services-dev
```

Comment:
I may be wrong, but if you have debian/compat set to 10, you should have debhelper (>= 10) here as well.

Reply:
9 works with compat 10 as well.